### PR TITLE
Initial 1.24.0 release notes

### PR DESCRIPTION
**Pull request by howardjohn**

### CODE REVIEW

**Review by bleggett — APPROVED** [review]
The notes look complete/correct to me and I don't see any omissions like we had with 1.23

File: content/en/news/releases/1.24.x/announcing-1.24/_index.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+[scaling](/blog/2024/ambient-vs-cilium/), [stabilizing](/blog/2024/ambient-reaches-beta/), and tuning ambient mode to be ready for prime time.
+
+On top of [countless changes since the Beta release](/news/releases/1.23.x/announcing-1.23/#ambient-ambient-ambient), Istio 1.24 comes with a number of enhancements to ambient mode:
+
+* New `status` messages are now written to a variety of resources, including `Services` and `AuthorizationPolicies`, to help understand the current state of the object.
+* Policies can now be attached directly to `ServiceEntry`s. Give it a try with a simplified [egress gateway](https://www.solo.io/blog/egress-gateways-made-easy/)!
+* A brand new, exhaustive, [troubleshooting guide](https://github.com/istio/istio/wiki/Troubleshooting-Istio-Ambient). Fortunately, a number of bug fixes in Istio 1.24 makes many of these troubleshooting steps no longer needed!
+* Many new bug fixes. In particular, edge cases around pods with multiple interfaces, GKE Intranode visibility, IPv4-only clusters, and many more have been improved.
```
Comment (kfaseela) [inline content/en/news/releases/1.24.x/announcing-1.24/_index.md:37]:
```suggestion
* Many bug fixes. In particular, edge cases around pods with multiple interfaces, GKE Intranode visibility, IPv4-only clusters, and many more have been improved.
```

File: content/en/news/releases/1.24.x/announcing-1.24/_index.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+### Improved retries
+
+Automatic [retries](/docs/concepts/traffic-management/#retries) has been a core part of Istio's traffic management functionality.
+In Istio 1.24, it gets even better.
+
+Previously, retries were exclusively implemented on the *client sidecar*.
+However, a common source of connection failures actually comes from communicating between the *server sidecar* and the server application,
+typically from attempting to re-use a connection the backend is closing.
+With this change, we are able to detect this case and retry on the server sidecar automatically.
```
Comment (kfaseela) [inline content/en/news/releases/1.24.x/announcing-1.24/_index.md:47]:
```suggestion
With the improved functionality, we are able to detect this case and retry on the server sidecar automatically.
```

File: content/en/news/releases/1.24.x/announcing-1.24/change-notes/index.md [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+description: Istio 1.24.0 release notes.
+publishdate: 2024-11-07
+release: 1.24.0
+weight: 10
+aliases:
+    - /news/announcing-1.24.0
+---
+
+{{< warning >}}
+This is an automatically generated rough draft of the release notes and has not yet been reviewed.
```
Comment (kfaseela) [inline content/en/news/releases/1.24.x/announcing-1.24/change-notes/index.md:14]: do we need this warning?

File: content/en/news/releases/1.24.x/announcing-1.24/upgrade-notes/index.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+---
+
+When upgrading from Istio 1.23.x to Istio 1.24.x, please consider the changes on this page.
+These notes detail the changes which purposefully break backwards compatibility with Istio 1.23.x.
+The notes also mention changes which preserve backwards compatibility while introducing new behavior.
+Changes are only included if the new behavior would be unexpected to a user of Istio 1.23.x.
+
+## Updated compatibility profiles
+
+To support compatibility with older versions, Istio 1.24 introduces a new 1.23 [compatibility profiles](/docs/setup/additional-setup/compatibility-versions/) and updates its other profiles to account for changes in Istio 1.24.
```
Comment (kfaseela) [inline content/en/news/releases/1.24.x/announcing-1.24/upgrade-notes/index.md:15]:
```suggestion
To support compatibility with older versions, Istio 1.24 introduces a new 1.23 [compatibility profile](/docs/setup/additional-setup/compatibility-versions/) and updates its other profiles to account for changes in Istio 1.24.
```

File: content/en/news/releases/1.24.x/announcing-1.24/change-notes/index.md [hunk 5]
```diff
@@ -0,0 +1,286 @@
...
+
+- **Added** support for `connection.sni` rule in `AuthorizationPolicy` applied to a waypoint.
+  ([Issue #52752](https://github.com/istio/istio/issues/52752))
+
+- **Updated** the redirection method used in Ambient from `TPROXY` to `REDIRECT`.
+  For most users, this should have no impact, but fixes a few compatibility issues with `TPROXY`.  ([Issue #52260](https://github.com/istio/istio/issues/52260)),([Issue #52576](https://github.com/istio/istio/issues/52576))
+
+## Traffic Management
+
+- **Promoted** Istio dual-stack support to Alpha
```
Comment (kfaseela) [inline content/en/news/releases/1.24.x/announcing-1.24/change-notes/index.md:58]: Should this move to the main page "What's new" ?

**Comment by kfaseela** [comment]
@mikemorris I am approving this, so that one of the RMs can work on a subsequent PR for any improvements.

**Review by kfaseela — APPROVED** [review]

File: content/en/news/releases/1.24.x/announcing-1.24/change-notes/index.md (hunk 3 above)
Comment (mikemorris) [reply content/en/news/releases/1.24.x/announcing-1.24/change-notes/index.md:14]: Pending removal in #15897

**Pull request merged**